js/controller.js

```diff
@@ -1749,6 +1752,24 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       }
     },
 
+    toggleMultiAudio: function () {
+      if (this.state.screenToShow == CONSTANTS.SCREEN.MULTI_AUDIO_SCREEN) {
+        this.closeScreen();
+        this.renderSkin();
```

Comment:
Ok, I have a task about toggleMultiAudio functionality. So It would be very comfortable for me to change it in next PR. Thank you! I will remember about your comment.